src/libraries/System.Net.Http.Json/tests/FunctionalTests/JsonContext/JsonContext.cs

```diff
@@ -13,11 +13,11 @@ internal partial class JsonContext : JsonSerializerContext
         private static JsonContext s_default;
         public static JsonContext Default => s_default ??= new JsonContext(new JsonSerializerOptions());
 
-        public JsonContext() : base(null)
+        public JsonContext() : base(null, null)
```

Comment:
This doesn't necessarily need to be fixed in this PR, but is there a reason this test doesn't use the source generator?

Reply:
It could - will explore it in a follow up.